packages/gatsby-theme-newrelic/README.md

```diff
@@ -667,6 +694,70 @@ const MyLayout = () => (
 );
 ```
 
+### `Dropdown`
+
+Used in combination with [`Dropdown.Toggle`](#dropdowntoggle), [`Dropdown.Menu`](#dropdownmenu), and [`Dropdown.MenuItem`](#dropdownmenuitem) to create dropdown menus.
+
+```js
+import { Dropdown } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop       | Type | Required | Default | Description                                                                    |
+| ---------- | ---- | -------- | ------- | ------------------------------------------------------------------------------ |
+| `align`    | enum | no       | "left"  | The position of the menu arrow. \*Must be either `left`, `right`, or `center`. |
+| `children` | node | yes      |         | Components used for the dropdown.                                              |
+
+```jsx
+import { DropDown, Button } from '@newrelic/gatsby-theme-newrelic';
+
+const Example = (
+  <Dropdown align="right">
+    <Dropdown.Toggle variant={Button.VARIANT.NORMAL}>Menu</Dropdown.Toggle>
+    <Dropdown.Menu>
+      <Dropdown.MenuItem>Item 1</Dropdown.MenuItem>
+      <Dropdown.MenuItem>Item 2</Dropdown.MenuItem>
+      <Dropdown.MenuItem>Item 3</Dropdown.MenuItem>
+    </Dropdown.Menu>
+  </Dropdown>
+);
+```
+
+#### `Dropdown.Toggle`
+
+Used within a [`Dropdown`](#dropdown) component to render a button that can toggle whether or not the dropdown menu is visible.
+
+**Props**
+
+| Prop       | Type | Required | Default | Description                                                          |
+| ---------- | ---- | -------- | ------- | -------------------------------------------------------------------- |
+| `size`     | enum | no       |         | The `size` prop for the underlying [`Button`](#button) component.    |
+| `variant`  | enum | yes      |         | The `variant` prop for the underlying [`Button`](#button) component. |
+| `children` | node | no       |         | Text or component used to render the toggle, in addition to an icon. |
```

Comment:
```suggestion
| `children` | node | no       |         | Content used to render the toggle |
```

Children can be whatever you want, there aren't really rules. I made this a bit more generic since there isn't really an "icon" you render unless you pull it in yourself.